modules/ocf_desktop/files/xsession/print-notify-listener

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+set -euo pipefail
+
+function clean_up {
+    jobs -pr | xargs pgrep -P | xargs sudo -u ocfbroker kill
+    exit 0
+}
+
+trap clean_up INT TERM EXIT
```

Comment:
INT and TERM are redundant since you already specified EXIT and this script is being run by bash